trestle/transforms/implementations/osco.py

```diff
@@ -379,6 +386,10 @@ def reviewed_controls(self) -> ReviewedControls:
     @property
     def result(self) -> Result:
         """OSCAL result."""
+        # perform result properties aggregation
+        if len(self.observations) > 0:
+            self._result_properties_list = TransformerHelper().remove_common_observation_properties(self.observations)
```

Comment:
this can just  be `if self.observations:`

Reply:
Fixed.